crates/astria-cli/src/types.rs

```diff
@@ -114,7 +114,8 @@ impl TryFrom<&ConfigCreateArgs> for RollupDeploymentConfig {
 pub struct RollupConfig {
     name: String,
     chain_id: String,
-    network_id: u64,
+    // NOTE - String here because
```

Comment:
Can we make this a bit more clear?

Reply:
haha yeah already fixed. Got in a rush :)